curriculum/teamwork-and-toolkit/schedule.md

```diff
@@ -2,99 +2,108 @@
+| 15:30   | [Git workshop][git-ws-120]           |                                    |
+| 17.30   | Pastoral care                        | Pastoral care on FAC20             |
+| 17:45   | Check-out                            |                                    | 
 
-[db-colour]: http://facresources.com/slides/design-burst-week1.html
-[db-colour-ws]: https://github.com/bobbysebolao/learn-css-variables
-[git-ws]: https://github.com/foundersandcoders/git-workflow-workshop-for-two
-[research-guide]: https://founders-and-coders.gitbook.io/coursebook/documents/spikes
-[technical-spikes]: ./week-1/spikes.md
+[spike-pres-60]: https://founders-and-coders.gitbook.io/coursebook/student-handbook/spikes#what-makes-a-good-research-presentation
+[user-manuals-30]: https://github.com/fac20/user-manuals
+[db-colour-30]: http://facresources.com/slides/design-burst-week1.html
+[db-colour-ws-60]: https://github.com/bobbysebolao/learn-css-variables
+[git-ws-120]: https://github.com/foundersandcoders/git-workflow-workshop-for-two
 
 ## Day 3
 
-| Time    | Activity                                 | Learning outcomes |
-| ------- | -----------------------------------------| ----------------- |
-| 09:45   | Check-in                                 |                   |
-| 10:00   | [Learn HTML forms][learn-forms]          | HTML forms        |
-| 11:00   | [Intro to project management][pm-slides] |                   |
-| 11:15   | [Project][projects]                      |                   |
-| _13:00_ | _Lunch_                                  |                   |
-| 14:00   | Project                                  |                   |
-| 17:00   | Speaker                                  |                   |
-
-[learn-forms]: https://github.com/oliverjam/learn-html-forms/
-[projects]: https://founders-and-coders.gitbook.io/coursebook/documents/projects
-[pm-slides]: https://hackmd.io/@fac/S1wGfV2M8#/
+| Time    | Activity                                     | Learning outcomes          |
+| ------- | -------------------------------------------- | -------------------------- |
+| 09:45   | Hello from Charlie, pastoral mentor          |                            |
+| 09:50   | Check-in                                     |                            |
+| 10:00   | [Learn HTML forms][learn-forms-60]           | HTML forms                 |
+| 11:00   | [Intro to project management][pm-slides-15]  | Managing tech projects     |
+| 11:15   | [Begin projects][projects]                   |                            |
+| _13:00_ | _Lunch_                                      |                            |
+| 14:00   | [Projects][projects]                         |                            |
+| 17:45   | Check-out                                    |                            |
 
+[learn-forms-60]: https://github.com/oliverjam/learn-html-forms/
+[pm-slides-15]: https://hackmd.io/@fac/S1wGfV2M8#/
+[projects]: ./project.md
```

Comment:
I don't think relative Markdown links work in Gitbook unfortunately (that's why we hard-coded the URL before)

Reply:
I think if it's linked in the sidebar maybe it's fine to not be linked in here

Reply:
Ah interesting! @Albadylic and I were wondering about why they're sometimes used and sometimes not.

Reply:
Just to clarify, would you like me to replace the relative link with https://founders-and-coders.gitbook.io/coursebook/curriculum/teamwork-and-toolkit/project ?

Reply:
I've added a space and changed relative paths to anchor links